### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Julita45**

### CODE REVIEW

File: fullTree.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+function drawTree(a) {
+	for (i=0; i<=a; i++) {
```
Comment (matwjablonski) [inline fullTree.js]: where is declaration for your iterator?

File: fullTree.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	
+	var star='';
+		for (j=0; j<=(a-i); j++) {
```
Comment (matwjablonski) [inline fullTree.js]: same as above

File: fullTree.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+		star += ' ';	
+		}	
+		for (k=0; k<=i*2; k++) {
```
Comment (matwjablonski) [inline fullTree.js]: same as above

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: fullTree.js [hunk 4]
```diff
@@ -1,17 +1,15 @@
...
+				star += ' ';	
+			}	
+			for (var k = 0; k <= i * 2; k++) {
+				star += '*';
+			}
 		console.log(star);
 	}
 }
-
 console.log(drawTree(5));
```
Comment (matwjablonski) [inline fullTree.js]: console.log here is unnecessary - you console.log your code in function ;)

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: fullTree.js [hunk 5]
```diff
@@ -9,7 +9,4 @@ function drawTree(a) {
 			}
 		console.log(star);
 	}
-}
-console.log(drawTree(5));
-console.log(drawTree(8));
-console.log(drawTree(20));
\ No newline at end of file
+}
```
Comment (matwjablonski) [inline fullTree.js]: ok, but why you remove your call for this function?

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by Julita45** [comment]
Ok, I get it ;)

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**